libdrizzle/statement.cc

```diff
@@ -392,7 +392,6 @@ drizzle_return_t drizzle_stmt_fetch(drizzle_stmt_st *stmt)
   drizzle_return_t ret= DRIZZLE_RETURN_OK;
   uint16_t column_counter= 0, current_column= 0;
   drizzle_row_t row;
-  drizzle_column_st *column;
```

Comment:
Do we need to move the declaration of the `drizzle_column_st ` into the for loop?